lib/reporting/authentication_report.rb

```diff
@@ -61,10 +61,16 @@ def as_tables
       ]
     end
 
-    def as_tables_with_options
+    def as_emailable_reports
```

Comment:
cc @Sgtpluck tagging you on this PR just as a heads up that I refactored this report you added